### PR TITLE
refactor: skeleton full token

### PR DESCRIPTION
**Pull request by MadCcc**
- refactor: skeleton full token
- refactor: skeleton full token

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/skeleton/style/index.tsx [hunk 1]
```diff
@@ -43,12 +46,12 @@ const genSkeletonElementAvatarSize = (size: number): CSSObject => ({
...
 const genSkeletonColor = (token: SkeletonToken): CSSObject => {
-  const { skeletonColor, skeletonToColor } = token;
+  const { color, colorGradientEnd } = token;
   return {
-    background: `linear-gradient(90deg, ${skeletonColor} 25%, ${skeletonToColor} 37%, ${skeletonColor} 63%)`,
+    background: `linear-gradient(90deg, ${color} 25%, ${colorGradientEnd} 37%, ${color} 63%)`,
     backgroundSize: '400% 100%',
     animationName: skeletonClsLoading,
-    animationDuration: '1.4s', // FIXME: magic
+    animationDuration: '1.4s',
```
Comment (zombieJ) [inline components/skeleton/style/index.tsx:54]: 提到下面去

File: components/skeleton/style/index.tsx [hunk 2]
```diff
@@ -43,12 +46,12 @@ const genSkeletonElementAvatarSize = (size: number): CSSObject => ({
 });
 
 const genSkeletonColor = (token: SkeletonToken): CSSObject => {
-  const { skeletonColor, skeletonToColor } = token;
+  const { color, colorGradientEnd } = token;
   return {
-    background: `linear-gradient(90deg, ${skeletonColor} 25%, ${skeletonToColor} 37%, ${skeletonColor} 63%)`,
+    background: `linear-gradient(90deg, ${color} 25%, ${colorGradientEnd} 37%, ${color} 63%)`,
```
Comment (zombieJ) [inline components/skeleton/style/index.tsx:51]: 提到下面去

File: components/skeleton/style/index.tsx (hunk 1 above)
Comment (MadCcc) [reply components/skeleton/style/index.tsx:54]: motionDurationLoading?

Comment (MadCcc) [reply components/skeleton/style/index.tsx:54]: 看了下还不一样= =

**Pull request merged**